androidenhancedvideoplayer/src/main/java/com/profusion/androidenhancedvideoplayer/components/EnhancedVideoPlayer.kt

```diff
@@ -14,15 +18,15 @@ import androidx.media3.ui.PlayerView
 
 private const val MAIN_PACKAGE_PATH_PREFIX = "android.resource://"
 
-@androidx.annotation.OptIn(UnstableApi::class)
 @Composable
 fun EnhancedVideoPlayer(
     resourceId: Int,
     useControls: Boolean = false,
     alwaysRepeat: Boolean = true,
     fullScreen: Boolean = true,
     playImmediately: Boolean = true,
-    soundOff: Boolean = true
+    soundOff: Boolean = true,
+    controlsDecoration: ControlsDecoration = ControlsDecoration()
```

Comment:
Instead of prop-drill `controlsDecoration`, what about receiving a `PlayerControls` from the parent that wants to customize?

Reply:
nit: I'd change the name to `controls` and receive a `PlayerControls`. The default behavior would be to create a `PlayerControls` with default arguments:

```suggestion
    controls: PlayerControls = PlayerControls()
```